dev/tests/verification/TestModule/ActionGroup/FunctionalActionGroup.xml

```diff
@@ -49,6 +49,24 @@
         <createData entity="simpleData" stepKey="createSimpleData"/>
         <grabTextFrom selector=".class" stepKey="grabTextData"/>
         <fillField stepKey="fill1" selector=".{$grabTextData}" userInput="$createSimpleData.field$"/>
+        <!--Invocation stepKey will not be appended in non stepKey instances-->
+        <click selector="$action0" stepKey="action0"/>
+        <fillField selector="$action1" stepKey="action1"/>
+        <!--Invocation stepKey will be appended to every instance of $stepKey in action-->
+        <executeJS function="$action3" stepKey="action3"/>
+        <magentoCLI command="$action4" arguments="&quot;stuffHere&quot;" stepKey="action4"/>
+        <generateDate date="$action5" format="H:i:s" stepKey="action5"/>
+        <formatMoney userInput="$action6" stepKey="action6"/>
```

Comment:
All of the above `$action#` should be `{$action#}`

Reply:
Fixed